### DISCUSSION TITLE
Error while installing npm packages, after cloning insomnia.

### DISCUSSION THREAD

**Discussion by rohitpatilok** [Kong/insomnia#8070] [Help] [answered]
I did the clone. My node version is 20.17.0. I get errors when installing the packages. I wanted to contribute, any help would be appreciated. Thank you!
![image](https://github.com/user-attachments/assets/bede7e7e-e96d-4aab-a5ad-ac5792c268a5)
![image](https://github.com/user-attachments/assets/70940bcf-8eb7-4ac6-b3ca-b88df211f445)

**Comment by gatzjames** [comment, marked as answer]
From the logs it looks like you are missing yarn in your environment.
You could install it by using npm add -g yarn and this error should go away

**Reply by rohitpatilok** [reply]
It worked! Thank you @gatzjames.
